### DISCUSSION TITLE
https://angular.love/pl/dynamiczna-konfiguracja-wykorzystanie-app-initializer?fbclid=IwZXh0bgNhZW0CMTEAAR2vgJk0simcsT3xAGn_t6SMfDULU01knQbQ-OS4fRgxIctiuxSokfgA1Lg_aem_p6TTV28G04VZH1tFB2YcyQ&utm_campaign=app-initializer&utm_medium=newsfeed&utm_source=facebook&utm_term=pl

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#327] [General]
# https://angular.love/pl/dynamiczna-konfiguracja-wykorzystanie-app-initializer?fbclid=IwZXh0bgNhZW0CMTEAAR2vgJk0simcsT3xAGn_t6SMfDULU01knQbQ-OS4fRgxIctiuxSokfgA1Lg_aem_p6TTV28G04VZH1tFB2YcyQ&utm_campaign=app-initializer&utm_medium=newsfeed&utm_source=facebook&utm_term=pl

Angular.love - ciekawostki oraz rozwiązania dla średnio-zaawansowanych oraz zaawansowanych developerów Angulara.

https://angular.love/pl/dynamiczna-konfiguracja-wykorzystanie-app-initializer?fbclid=IwZXh0bgNhZW0CMTEAAR2vgJk0simcsT3xAGn_t6SMfDULU01knQbQ-OS4fRgxIctiuxSokfgA1Lg_aem_p6TTV28G04VZH1tFB2YcyQ&utm_campaign=app-initializer&utm_medium=newsfeed&utm_source=facebook&utm_term=pl

**Comment by freoff** [comment]
warto dodac ze wszystkie tokeny APP-INITIALIZER SA uruchamiane jednoczesnie, cos jak promiseAll, wiec jesli bedzie zaleznosc miedzy poszczegolnymi konfiuracjami lub czymkolwiek co bedzie tam inicjalizowane  musza sie znalesc w jednej faktorce. i wykonane w odpowiedniej kolejnosci, lub dodac jakies bardziej zawansowane rozwiazanie w zaleznosci od potrzeb.
